client/src/App.tsx

```diff
@@ -22,8 +23,8 @@ function App(): JSX.Element {
             <SocketProvider>
               <NavBar />
               <Switch>
-                <Route exact path="/login" component={Login} />
-                <Route exact path="/signup" component={Signup} />
+                <AuthProtectedRoute exact path="/login" component={Login} />
```

Comment:
I don't think we need two seperate protected routes, let's keep only  one of them :)